nondex-maven-plugin/src/it/systematic-it/SystematicTest.java

```diff
@@ -0,0 +1,51 @@
+
+/*
+The MIT License (MIT)
+Copyright (c) 2015 Alex Gyori
+Copyright (c) 2015 Owolabi Legunsen
+Copyright (c) 2015 Darko Marinov
+Copyright (c) 2015 August Shi
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+package edu.illinois.nondex.it;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+import static org.junit.Assert.*;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.junit.Test;
+
+/**
+ * Unit test for simple App.
+ */
+public class SysteamticTest {
+
+    @Test
+    public void test() {
+        Set<Integer> s = new HashSet<Integer>();
+        s.add(2);
+        s.add(3);
+	s.add(4);
```

Comment:
I have moved it to the correct directly, mvn install/verify freezes now.  I believe it is because what I have added to pom:
<nondexMode> SYSTEMATIC </nondexMode>
When I remove this the test runs but it fails since it is in Full mode.